zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,409 @@
+        if task.allow_image_completing():
+            pool, img_name = task.dstPath.split('/')
+            ioctx = self.get_ioctx(pool)
+            complete_upload(task, ioctx)
+
+    def get_upload_param(req_header):
+        # type: (dict[str, str]) -> UploadParam
+
+        def get_long_field(key, default=None):
+            v = req_header.get(key, default)
+            try:
+                lv = long(v)
```

Ruff (0.13.1)

Undefined name `long`

(F821)